roles/openshift_aws/defaults/main.yml

```diff
@@ -46,6 +47,45 @@ openshift_aws_ami_tags:
 openshift_aws_s3_mode: create
 openshift_aws_s3_bucket_name: "{{ openshift_aws_clusterid }}-docker-registry"
 
+openshift_aws_dns_dict:
```

Comment:
I don't see the sense in making this one monolithic dictionary.

This seems like going overboard for code-reuse.